javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,80 @@
+//Alireza hassanloo
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1979;
+const yearFuture = 2037;
+const age = yearFuture - yearOfBirth;
+
+//first way
+console.log(`I will be ${age} years old in ${yearFuture}`);
+//second way
+console.log("I will be", age, "years old in", yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const dogHumanYear = dogYear * 7;
+let shouldShowResultInDogYears;
+
+if (shouldShowResultInDogYears) {
+  console.log(
+    "Your dog will be ",
+    dogYear,
+    "human years old in ",
+    dogYearFuture
+  );
+} else {
+  console.log(
+    "Your dog will be ",
+    dogHumanYear,
+    "human years old in ",
+    dogYearFuture
+  );
+}
+
+//Housey pricey (A house price estimator)
+
+const wideOfHouse = 5;
+const deepOfHouse = 11;
+const highOfHouse = 8;
+
+const volumeInMeters = wideOfHouse * deepOfHouse * highOfHouse;
+const gardenSizeInM2 = 70;
+const priceWillPaid = 2_500_000;
```

Comment:
This is a bit confusing price format with the `_` signs, Could you keep it as a whole number? Also this is the price of the house for Peter, but the other variables are from Julia's house.
You also need to write the code so it solves both cases - for Julia and for Peter.